packaging/package/manifest.py

```diff
@@ -6,8 +6,12 @@
 
 from collections import OrderedDict
 from six import BytesIO
+from .version import __version__
 
 manifest_line_length = 70
+MANIFEST_VERSION = "Manifest-Version"
+VENDOR_KEY = "Created-By"
+SDK_NAME = "Tableau Connector SDK "
```

Comment:
Should this be the SDK or something like `Tableau Packaging Tool`? I'm asking because your example shows the version number of `packaging`, not the SDK (which uses dates in its changelog).

Reply:
yeah, you are right.

Reply:
added a new commit to this PR.